protostar/testing/cairo_cheatcodes/declare_contract_cairo_cheatcode.py

```diff
@@ -0,0 +1,102 @@
+import asyncio
+from dataclasses import dataclass
```

Comment:
I would keep the old names of cheatcodes
```
declare_cairo_cheatcode
deploy_cairo_cheatcode
prepare_cairo_cheatcode
deploy_contract_cairo_cheatcode
```

Reply:
done